matching-service/src/model/user.ts

```diff
@@ -2,7 +2,8 @@ export type TDifficulty = "easy" | "medium" | "hard";
 export type SelectedDifficultyData = {[difficulty in TDifficulty] : boolean};
 
 interface User {
-    userToken : string;
+    id : string;
+    email : string;
```

Comment:
are you still using email here? (like for console logging and all that)

Reply:
i was gonna ask yall this but yall look very busy now haha
its possible to match users by id but console log with email so we shld decide what we what to use for console log since its way easier to read with email but both works equally well

Reply:
anyways i was planning to have a commit to remove all the redundant code in the future so if we r logging with user id i'll remove this in that commit

Reply:
i think just leave it there? allow us to more easily distinguish the users and who is who

thinking something like `userID <email>` as a queue entry